.github/workflows/test.yml

```diff
@@ -1,44 +1,28 @@
-name: Test
+name: Lint and test
```

Comment:
i'm assuming the advantage of merging the job is deduplicating rules, right? we want them to run on the same events and conditions while centralizing the config? should i assume it's also slightly faster?

Reply:
> i'm assuming the advantage of merging the job is deduplicating rules, right? we want them to run on the same events and conditions while centralizing the config?

Yes, exactly. We could always look at different config strategies like sharing workflows if we want to separate them. For now, though, consolidation seems like the most straightforward approach 

> should i assume it's also slightly faster?

Great question. It's definitely faster since the cairo lint and cairo test both require the cairo submodule (and that takes some time to install)